tasking/pulp/tasking/scheduler.py

```diff
@@ -258,7 +258,7 @@ def tick(self):
                           % {'celerybeat_name': self.celerybeat_name})
             ret = self.call_tick(self, self.celerybeat_name)
         except TaskLock.DoesNotExist:
-            TaskLock.objects.get(name=self.celerybeat_name, timestamp__lte=old_timestamp).delete()
+            TaskLock.objects.filter(name=self.celerybeat_name, timestamp__lte=old_timestamp).delete()
```

Comment:
@goosemania I thikn you are right that the cleanup on line 261 is supposed to remove any locks that were created by other machines. So the 'lock' field should be filtered on instead of the 'name'.